maven-model-builder/src/main/java/org/apache/maven/model/path/ProfileActivationFilePathInterpolator.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.maven.model.path;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.maven.model.profile.ProfileActivationContext;
+import org.codehaus.plexus.interpolation.InterpolationException;
+
+/**
+ * Interpolates path for {@link org.apache.maven.model.ActivationFile}.
+ *
+ * @author Ravil Galeyev
+ */
+public interface ProfileActivationFilePathInterpolator
```

Comment:
why do we need this interface? A class seems like enough.

Reply:
I tried to keep the same code style. I took `UrlNormalizer` as an example.
Anyway, I made `ProfileActivationFilePathInterpolator` a class. If an interface looks better, let me know, I'll revert :)